db.py

```diff
@@ -40,7 +44,8 @@ def migrate():
         for migration in sorted(list(migrations - existing)):
             with c.cursor() as curs:
                 print("Running migration %s" % migration)
-                sql = open("./migrations/%s.sql" % migration).read()
+                filename = "%s/%s.sql" % (migrations_directory, migration)
+                sql = open(filename).read()
```

Comment:
I'm not 100% sure about what happens to the file object that gets `open`ed here if an error occurs. It may get cleaned up by the enclosing `with`, but I'm concerned that might not be the case. How do you feel about:

```python
with c.cursor() as curs, open(filename) as sql_file:
    sql = sql_file.read()
    # Other stuff
```

The line where  the `filename` is built can go above the `with` in this case.